Solutions/SharpArch.Domain/DomainModel/BaseObject.cs

```diff
@@ -55,8 +46,8 @@ public override bool Equals(object obj)
                 return true;
             }
 
-            return compareTo != null && this.GetType().Equals(compareTo.GetTypeUnproxied()) &&
-                   this.HasSameObjectSignatureAs(compareTo);
+            return compareTo != null && GetType().Equals(compareTo.GetTypeUnproxied()) &&
```

Comment:
:( Does the resharper code style not work anymore? or do you not have resharper? Things should have been picked up from https://github.com/sharparchitecture/Sharp-Architecture/blob/master/Solutions/SharpArch.DotSettings

The one I remember, and saw in this pull request are the use of 'this' and always having parenthesis around single line blocks:

     if(something)
     {
         return x;
    }

instead of 
    if(something) return x;